docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -99,13 +99,13 @@ def validate(self):
         if self.settings.os not in ["Linux", "FreeBSD", "Macos"]:
             raise ConanInvalidConfiguration(f"{self.ref} is not supported on {self.settings.os}.")
 
-    # if another tool than the compiler or autotools is required to build the project (pkgconf, bison, flex etc)
+    # if a tool other than the compiler or autotools is required to build the project (pkgconf, bison, flex etc)
     def build_requirements(self):
         # only if we have to call autoreconf
-        self.tool_requires("libtool/x.y.z")
+        self.tool_requires("libtool/2.4.7")
         # only if upstream configure.ac relies on PKG_CHECK_MODULES macro
         if not self.conf.get("tools.gnu:pkg_config", check_type=str):
-            self.tool_requires("pkgconf/x.y.z")
+            self.tool_requires("pkgconf/2.1.0")
```

Comment:
please, revert to x.y.z. Otherwise, people will not check the latest version available when using the template.

Reply:
Ok. The Meson recipe template already uses real versions, though. Should I blank these out as well?